packages/siwe/lib/client.test.ts

```diff
@@ -201,15 +179,17 @@ describe(`Unit`, () => {
       issuedAt: '2022-01-27T17:09:38.578Z',
       chainId: 1,
       expirationTime: '2100-01-07T14:31:43.952Z',
-    });
+    };
+    const msg = new SiweMessage(params);
     const signature = await wallet.signMessage(msg.toMessage());
-    const result = await (msg as any).verify({ signature });
-    expect(result.success).toBeTruthy();
+    const { domain, nonce } = params;
+    const result = await (msg as any).verify({ signature, domain, nonce, time: (new Date()).toLocaleString() });
```

Comment:
People will have to go thru a similar flow to this instead of passing flags to skip nonce/domain verification. This is probably acceptable, but just want to consciously make this decision